pvlib/par.py

```diff
@@ -0,0 +1,72 @@
+"""
+Photosynthetically Active Radiation (PAR) module.
+Utilities found here are specially interesting for agrivoltaic systems.
+"""
+
+from pvlib.tools import cosd, sind
+
+
+def spitters_relationship(solar_zenith, global_diffuse_fraction):
+    r"""
+    Derive the diffuse fraction of photosynthetically active radiation (PAR)
+    respect to the global radiation diffuse fraction.
+
+    The relationship is based on the work of Spitters et al. (1986) [1]_.
+
+    Parameters
+    ----------
+    solar_zenith : numeric
+        Solar zenith angle in degrees :math:`^{\circ}`.
+
+    global_diffuse_fraction : numeric
+        Fraction of the global radiation that is diffuse. Unitless.
+
+    Returns
+    -------
+    par_diffuse_fraction : numeric
+        Photosynthetically active radiation in W/m^2.
```

Comment:
@kandersolar Do we have a standard format for specifying units? I've seen at least three different methods in our documentation:
- ... active radiation in W/m^2.
- ... active radiation. W/m^2.
- ... active radiation. [W/m^2]

Reply:
This question just came up elsewhere as well, with an additional contender of ``` :math:`W/m^2` ```. 

I don't think we have anything that rises to the level of a "standard", but I personally favor `[W/m^2]` unless there is good reason for something else.  Reasons:
- It makes sense to me for the unit to be visually separated from the text description (i.e. not in the same sentence) as it may prevent ambiguity when other quantities are mentioned in the description.  So "... radiation in W/m^2" is out.
- The `:math:` version results in bold font that looks a little jarring to me at the end of each line.
- I think (but am not 100% confident) that `[W/m^2]` is the most common form currently in pvlib, so consistency supports keeping it.

If I am being honest, the main reason for me might just be that I am used to [W/m^2].  Anyway, that's my two (three?) cents!

Reply:
I agree with the reasoning. There's still another alternative, which is using the ascii character superscript two:
`[W/m²]`
I personally prefer this one, to me it's like the caret adds visual noise.